### PR TITLE
List CUDA toolkit issue as known problem

### PR DESCRIPTION
**Pull request by prckent**
## Proposed changes

Ensure this is not forgotten. CI uses 11.2 for offload builds to avoid this issue.

## What type(s) of changes does this code introduce?

- Documentation changes

### Does this introduce a breaking change?

- No

## What systems has this change been tested on?

NA

## Checklist

- Yes. Documentation has been added (if appropriate)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+## [Unreleased]
+
+# Known problems
+
+* When offload builds are compiled with CUDA toolkit versions above 11.2 (tested
+  11.3-11.8) using LLVM15, multideterminant tests and functionality will fail,
+  seemingly due to an issue with the toolkit. This is discussed in
+  https://github.com/llvm/llvm-project/issues/54633 . All other functionality
+  appears to work as expected. As a workaround, the CUDA toolkit 11.2 can be
+  used. The actual CUDA drivers can be more recent.
```
Comment (ye-luo) [inline CHANGELOG.md:14]: NVIDIA drivers not CUDA drivers

Comment (prckent) [reply CHANGELOG.md]: Addressed

**Comment by ye-luo** [comment]
Test this please

**Review by ye-luo — APPROVED** [review]

**Comment by prckent** [comment]
Test this please

**Comment by williamfgc** [comment]
Test this please after resetting nitrogen rocm

**Comment by williamfgc** [comment]
Test this please after shutdown

**Pull request merged**